modules/masterbar/masterbar.php

```diff
@@ -563,6 +563,20 @@ public function add_my_sites_submenu( $wp_admin_bar ) {
 			) );
 		}
 
+		// Site Preview
+		if ( is_admin() ) {
+			$wp_admin_bar->add_menu( array(
+				'parent' => 'blog',
+				'id'     => 'site-preview',
+				'title'  => __( 'Site Preview' ),
```

Comment:
Oh right, guarding against malicious translations and plugins that filter them totally makes sense. Since we should be doing that in several other places in Masterbar's code, I'm going to spin up a separate PR that fixes that. Thanks for pointing this out! :heart: